src/test/test_verify.cpp

```diff
@@ -315,9 +323,8 @@ TEST_SECTION("suricata", "xdp_filter.o", "xdp")
 
 // Test some programs that ought to fail verification.
 //
-// False negative: exposing kernel pointers to user-mode apps
-// This should be changed to TEST_SECTION_FAIL once fixed.
```

Comment:
`TEST_SECTION_FAIL` is for false positives; added `TEST_SECTION_REJECT` macro for negatives.